README.md

```diff
@@ -54,10 +54,13 @@ If you are using the GDS development virtual machine then the application will b
 
 ## Running the test suite
 
-Before you can run the test suite you'll need the [govuk-content-schemas]
-repository locally. See
-[`lib/govuk_content_schema_examples.rb`][content_schema_examples] for more
-details.
+Before you can run the test suite you'll need the [govuk-content-schemas] repository locally. See [`lib/govuk_content_schema_examples.rb`][content_schema_examples] for more details.
```

Comment:
Would be cool if we could use the gem https://github.com/alphagov/govuk_schemas instead of requiring another repo locally.

Reply:
That'd be good indeed. I think it needs to be in another pull request as I only removed the line breaks to make the readme formatting a bit more consistent.